components/style/build.rs

```diff
@@ -43,19 +43,7 @@ fn main() {
         .env("PYTHONPATH", &mako)
         .env("TEMPLATE", &template)
         .env("PRODUCT", product)
-        .arg("-c")
-        .arg(r#"
-import os
-import sys
-from mako.template import Template
-from mako import exceptions
-try:
-    template = Template(open(os.environ['TEMPLATE'], 'rb').read(), input_encoding='utf8')
-    print(template.render(PRODUCT=os.environ['PRODUCT']).encode('utf8'))
-except:
-    sys.stderr.write(exceptions.text_error_template().render().encode('utf8'))
-    sys.exit(1)
-"#)
```

Comment:
This change is not _mandatory_ in the PR, but it felt natural in line with the thinking to "avoid multiple languages in the same file" (can't find the reference now...). Maintaining Python code is also easier when it has a proper `.py` suffix etc.